### PR TITLE
Replaces Mice w/ Rats

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Currently literally only a visual change. Might do some more with this soon.
Rats are now slightly more robust, capable of taking a single hit from a weapon, give more meat, and there is more of them in the infestation event. Deletes rat toxin.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!wip

**Comment by AmoryBlaine** [comment]
vewy nice

File: strings/phobia.json [hunk 1]
```diff
@@ -121,6 +121,8 @@
 		"roach",
 		"mouse",
```
Comment (Sindorman) [inline strings/phobia.json]: remove this and mice then

Comment (ParadoxSpace) [reply strings/phobia.json]: Isn't this just for people saying it?

Comment (MarinaGryphon) [reply strings/phobia.json]: mouse/mice should stay, add rat as well

File: code/__defines/_macros.dm [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 #define isliving(A) istype(A, /mob/living)
 
-#define ismouse(A) istype(A, /mob/living/simple_animal/mouse)
+#define ismouse(A) istype(A, /mob/living/simple_animal/rat)
```
Comment (skull132) [inline code/__defines/_macros.dm]:
```suggestion
#define israt(A) istype(A, /mob/living/simple_animal/rat)
```
Code should never tell lies.

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 3]
```diff
@@ -81,7 +81,7 @@
 		src.set_dir(turn(src.dir, 90))
 		return
 	else
-		if(istype(usr,/mob/living/simple_animal/mouse))
+		if(istype(usr,/mob/living/simple_animal/rat))
```
Comment (skull132) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm]:
```suggestion
		if(israt(usr))
```

File: code/modules/hydroponics/seed.dm [hunk 4]
```diff
@@ -107,7 +107,7 @@
 		return
 
 	if(!istype(target))
-		if(istype(target, /mob/living/simple_animal/mouse))
+		if(istype(target, /mob/living/simple_animal/rat))
```
Comment (skull132) [inline code/modules/hydroponics/seed.dm]:
```suggestion
		if(israt(target))
```

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 5]
```diff
@@ -19,7 +19,7 @@
 	response_help  = "pets"
 	response_disarm = "gently pushes aside"
 	response_harm   = "kicks"
-	//var/mob/living/simple_animal/mouse/movement_target
+	//var/mob/living/simple_animal/rat/movement_target
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 6]
```diff
@@ -30,21 +30,21 @@
...
-	var/mob/living/simple_animal/mouse/mousetarget = null
+	var/mob/living/simple_animal/rat/mousetarget = null
 	seek_speed = 5
 	pass_flags = PASSTABLE
 	possession_candidate = 1
 	butchering_products = list(/obj/item/stack/material/animalhide/cat = 2)
 
 /mob/living/simple_animal/cat/think()
-	//MICE!
+	//THEM RATS!
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]:
```suggestion
	//!
```

**Comment by ToasterStrudes** [comment]
Nanako rolls in their grave

**Comment by ParadoxSpace** [comment]
!review

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 7]
```diff
@@ -30,21 +29,21 @@
...
-	var/mob/living/simple_animal/mouse/mousetarget = null
+	var/mob/living/simple_animal/rat/mousetarget = null
 	seek_speed = 5
 	pass_flags = PASSTABLE
 	possession_candidate = 1
 	butchering_products = list(/obj/item/stack/material/animalhide/cat = 2)
 
 /mob/living/simple_animal/cat/think()
-	//MICE!
+	//!
```
Comment (Arrow768) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]: That should be removed as it serves no purpose

File: code/modules/mob/living/simple_animal/friendly/rat.dm [hunk 8]
```diff
@@ -1,311 +1,326 @@
...
+				if (diff > 600)
+					squeals++
+					last_squealgain = world.time
+
+	else
+		if ((world.time - timeofdeath) > decompose_time)
+			dust()
+
+/mob/living/simple_animal/rat/Destroy()
+	SSmob.all_mice -= src
```
Comment (Arrow768) [inline code/modules/mob/living/simple_animal/friendly/rat.dm]: Still references mice

File: code/modules/mob/living/simple_animal/friendly/rat.dm [hunk 9]
```diff
@@ -1,311 +1,326 @@
...
+	if (body_color == "white")
+		holder_type = /obj/item/weapon/holder/rat/white
+	if (body_color == "hooded")
+		holder_type = /obj/item/weapon/holder/rat/hooded
+	if (body_color == "irish")
+		holder_type = /obj/item/weapon/holder/rat/irish
+
+	//verbs += /mob/living/simple_animal/rat/proc/squeak
+	//verbs += /mob/living/simple_animal/rat/proc/squeak_soft
+	//verbs += /mob/living/simple_animal/rat/proc/squeak_loud(1)
```
Comment (Arrow768) [inline code/modules/mob/living/simple_animal/friendly/rat.dm]: Remove the dead code here

File: code/modules/mob/living/simple_animal/friendly/rat.dm [hunk 10]
```diff
@@ -1,311 +1,326 @@
...
+
+	SSmob.all_mice += src
```
Comment (Arrow768) [inline code/modules/mob/living/simple_animal/friendly/rat.dm]: Still references mice

File: code/modules/mob/living/simple_animal/friendly/rat.dm [hunk 11]
```diff
@@ -1,311 +1,326 @@
...
+
+//Plays a sound.
+//This is triggered when a mob steps on an NPC mouse, or manually by a playermouse
+/mob/living/simple_animal/rat/proc/squeak(var/manual = 1)
+	if (stat == CONSCIOUS)
+		if (squeakcooldown > world.time)
+			return
+		squeakcooldown = world.time + 2 SECONDS
+
+		playsound(src, 'sound/effects/mousesqueek.ogg', 70, 1)
```
Comment (Arrow768) [inline code/modules/mob/living/simple_animal/friendly/rat.dm]: You could rename the soundfiles while you are at it.

File: code/modules/mob/living/simple_animal/friendly/rat.dm [hunk 12]
```diff
@@ -1,311 +1,326 @@
...
+//Triggered manually, when a mouse dies, or rarely when its stepped on
+/mob/living/simple_animal/rat/proc/squeak_loud(var/manual = 0)
+	if (stat == CONSCIOUS)
+		if (squeakcooldown > world.time)
+			return
+
+		squeakcooldown = world.time + 4 SECONDS
+
+		if (squeals > 0 || !manual)
+			playsound(src, 'sound/effects/creatures/mouse_squeak_loud.ogg', 50, 1)
```
Comment (Arrow768) [inline code/modules/mob/living/simple_animal/friendly/rat.dm]: You could rename the soundfiles while you are at it.

File: code/modules/mob/living/simple_animal/friendly/rat.dm [hunk 13]
```diff
@@ -1,311 +1,326 @@
...
+		K.absorb(src)
+	..()
+
+/mob/living/simple_animal/rat/death()
+	layer = MOB_LAYER
+	if (stat != DEAD && (ckey || prob(50)))
+		squeak_loud(0)//deathgasp
+
+	if(client)
+		client.time_died_as_mouse = world.time
```
Comment (Arrow768) [inline code/modules/mob/living/simple_animal/friendly/rat.dm]: References mice

File: code/modules/mob/living/simple_animal/friendly/rat.dm [hunk 14]
```diff
@@ -1,311 +1,326 @@
...
+
+	SSmob.all_mice -= src
+
+	..()
+
+/mob/living/simple_animal/rat/dust()
+	..(anim = "dust_[body_color]", remains = /obj/effect/decal/remains/mouse, iconfile = 'icons/mob/npc/rat.dmi')
```
Comment (Arrow768) [inline code/modules/mob/living/simple_animal/friendly/rat.dm]: `/obj/effect/decal/remains/mouse`

File: code/modules/mob/living/simple_animal/friendly/rat.dm [hunk 15]
```diff
@@ -1,311 +1,326 @@
...
+	var/maxSqueals = 2//SPAM PROTECTION
+	var/last_squealgain = 0// #TODO-FUTURE: Remove from life() once something else is created
+	var/squeakcooldown = 0
+	pass_flags = PASSTABLE
+	speak_chance = 3
+	turns_per_move = 5
+	see_in_dark = 6
+	maxHealth = 10
+	health = 10
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/mice
```
Comment (Arrow768) [inline code/modules/mob/living/simple_animal/friendly/rat.dm]: `/obj/item/weapon/reagent_containers/food/snacks/meat/mice`

File: code/modules/mob/living/simple_animal/friendly/rat.dm [hunk 16]
```diff
@@ -1,311 +1,326 @@
...
+	icon_rest = "rat_gray_sleep"
+	can_nap = 1
+	speak = list("Squeek!","SQUEEK!","Squeek?")
+	speak_emote = list("squeeks","squeeks","squiks")
+	emote_hear = list("squeeks","squeaks","squiks")
+	emote_see = list("runs in a circle", "shakes", "scritches at something")
+	var/soft_squeaks = list('sound/effects/creatures/mouse_squeaks_1.ogg',
+	'sound/effects/creatures/mouse_squeaks_2.ogg',
+	'sound/effects/creatures/mouse_squeaks_3.ogg',
+	'sound/effects/creatures/mouse_squeaks_4.ogg')
```
Comment (Arrow768) [inline code/modules/mob/living/simple_animal/friendly/rat.dm]: You could rename the soundfiles while you are at it.

**Comment by ParadoxSpace** [comment]
!review

**Pull request merged**